ccm_web/client/src/components/InlineTextEdit.tsx

```diff
@@ -85,20 +83,29 @@ function InlineTextEdit (props: InlineTextEditProps): JSX.Element {
     }
   }
 
+  const renderEditButton = (): JSX.Element => {
+    if (!props.isSaving) {
+      return (<EditIcon className={classes.editIcon} fontSize='inherit' onClick={toggleEdit}/>)
+    } else {
+      return (<EditIcon className={classes.editIconDisabled} fontSize='inherit'/>)
+    }
+  }
+
   return (
     <form className={classes.root} noValidate autoComplete="off">
+      Is Saving &quot;{props.isSaving.toString()}&quot;
       <Grid container className={classes.inputArea}>
         <Grid item xs={8} sm={8}>
           <TextField className={classes.inputRow} aria-readonly={false} onClick={toggleEdit} inputProps={{ style: { fontSize: 24 } }} ref={textInput} id="standard-basic" placeholder={props.placeholderText} value={tempTextValue} onKeyDown={(e) => keyPress(e.code)} onChange={(e) => setTempTextValue(e.target.value)} disabled={!isEditing}/>
         </Grid>
```

Comment:
More like so?

![image](https://user-images.githubusercontent.com/17617998/124941907-29f8ca80-dfd9-11eb-8df8-a506e677936e.png)

Reply:
![image](https://user-images.githubusercontent.com/17617998/124942011-3da43100-dfd9-11eb-8ba6-f81162571a93.png)

Reply:
Yeah, something like that. Can we fix it so that the combined length of the field and the buttons stays mostly even with the feature tiles below? Or alternatively, is there a way to make it adjust in size as you type (with a minimum and maximum)? Latter seems more complicated, unless it's built into `TextField`.

Reply:
That was my thought too, the issue here is there are 2 buttons for each collection of features, but each row there is sized to fit 3, so if a 3rd feature gets added it will fill that area to the right and it will line up with the input.  I suppose that makes my argument more of a philosophical one than a real one since we're not adding a 3rd feature to any section.

Reply:
There might be something you can do with `Grid` to use 2/3's of the space. I think even if we added a third feature, we wouldn't want it to be that long.

Reply:
So it's using 2/3 of the space for the input, and 1/3 for the buttons.  
When you're editing seems like maybe when horizontal space is most important, and that's when there is more to display (the buttons)
Not sure what is ideal.
The original size I think was what the mockup looked like, but I agree that when I'm just messing around I often have names longer than the input, so not sure what is best..

Reply:
Maybe we could just make it a little larger than what it is now, using a fixed length, and make sure it looks okay in different screen sizes, and then push off the discussion for later? Or we could leave it as is for now. I don't think this is the last time we'll think about text field styling.

Reply:
I'll just leave it for now I guess.  Not really related to hooking up the backend anyways.

The only change required to make it behave as above is to add the attribute `fullWidth={true}` to TextField